CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.1.0
+- Uplifted spring-boot version to 2.7.5 from 2.0.1.RELEASE
```

Comment:
I recommend keeping the "... from <version>  to <version> .." 
This just gets harder to read when mixing formats.

Reply:
Updated the change-log file